service/exceptionmonitor/helper/client/client.go

```diff
@@ -0,0 +1,38 @@
+package client
+
+import (
+	"os"
+
+	"github.com/labring/sealos/service/exceptionmonitor/api"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func InitClient() error {
+	kubeconfigPath := "/home/nonroot/kubeconfig/kubeconfig"
```

Comment:
What is this? Get kubernetes.client through k8s.io/client-go/rest.InClusterConfig()